starter-code/server.js

```diff
@@ -23,8 +25,10 @@ app.get('/new', function(request, response) {
 
 app.get('/articles', function(request, response) {
   // REVIEW: This query will join the data together from our tables and send it back to the client.
-  // TODO: Write a SQL query which joins all data from articles and authors tables on the author_id value of each
-  client.query(``)
+  //Done! TODO: Write a SQL query which joins all data from articles and authors tables on the author_id value of each
+  client.query1(`SELECT * FROM articles
+     JOIN authors ON articles.article_id=articles.author_id;`)
```

Comment:
We should be using the author id from the articles table to connect the two tables